src/CreatePaletteForm/CreatePaletteForm.js

```diff
@@ -41,17 +41,33 @@ export class CreatePaletteForm extends Component {
     this.setState({ name: "" })
   }
 
+  handleSaveEdits = async () => {
+    const { currentPalette, currentPaletteId, setSelectedPalettes, clearSelectedPaletteId, setRandomPalette } = this.props;
+    const paletteToEdit = {
+      color1: currentPalette[0].hexCode,
+      color2: currentPalette[1].hexCode,
+      color3: currentPalette[2].hexCode,
+      color4: currentPalette[3].hexCode,
+      color5: currentPalette[4].hexCode
+    }
+    await editPalette(paletteToEdit, currentPaletteId.id)
+    const updatePalettes = await getSelectedPalettes(currentPaletteId.projectId);
+    setSelectedPalettes(updatePalettes);
+    clearSelectedPaletteId();
+    setRandomPalette();
+  }
+
   render() {
-    const { currentProjects } = this.props;
+    const { currentProjects, currentPaletteId } = this.props;
     let displayProjects = currentProjects.map(project => {
       return <option key={project.name} value={project.name}>{project.name}</option>
     })
     return (
       <section className="CreatePalette_section">
-        <div>
-          <button>Save Changes</button>
-        </div>
-        <form className="CreatePalette_form">
+        {currentPaletteId && <div>
+          <button onClick={this.handleSaveEdits}>{`Save Changes for ${currentPaletteId.name}`}</button>
```

Comment:
I think it would be nice to add an additional button, so if the user changes their mind on revising. They can go back to the creating a new palette form instead.